utils/common/src/main/java/org/apache/brooklyn/util/stream/StreamGobbler.java

```diff
@@ -23,17 +23,20 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.io.PrintStream;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import org.slf4j.Logger;
 
 public class StreamGobbler extends Thread implements Closeable {
-    
+
+    private static char REPLACEMENT_CHARACTER = 0xfffd;
```

Comment:
@algairim Make this final.

Reply:
Addressed in ba780b1acac6c42b4d4f5cc8aa0d063339869847.